lib/libipmeta.h

```diff
@@ -495,6 +531,25 @@ int ipmeta_provider_maxmind_get_iso2_list(const char ***countries);
 int ipmeta_provider_maxmind_get_country_continent_list(
   const char ***continents);
 
+/** Convenience function to retrieve a list of ISO 2 character country codes
+ *
+ * @param countries[out]   The provided pointer is updated to point to an
+ *                         array of 2 character country code strings
+ * @return the number of elements in the array
+ */
+int ipmeta_provider_maxmind_v2_get_iso2_list(const char ***countries);
```

Comment:
are these actually implemented? i can't find them anywhere.
if they are not, then please remove these declarations.